### PR TITLE
[DOCFIX] Update Chinese HDFS Docs

### PR DESCRIPTION
**Pull request by wuzhenhua01**
### What changes are proposed in this pull request?

Update Chinese HDFS Docs.

### Why are the changes needed?

The Chinese ufs/HDFS document is not updated with the latest changes, this PR synchronizes these updates

### Does this PR introduce any user facing changes?

NO

### CODE REVIEW

File: docs/cn/ufs/HDFS.md [hunk 1]
```diff
@@ -15,22 +15,25 @@ priority: 3
...
-注意，在编译源码包的时候，默认的Alluxio二进制包适用于HDFS `2.7.3`，若使用其他版本的Hadoop，需要指定正确的Hadoop版本，并且在Alluxio源码目录下运行如下命令：
+注意，在编译源码包的时候，默认的Alluxio二进制包适用于HDFS `3.3.1`，若使用其他版本的Hadoop，需要指定正确的Hadoop版本，并且在Alluxio源码目录下运行如下命令：
 
 ```console
-$ mvn install -P<YOUR_HADOOP_PROFILE> -DskipTests
+$ mvn install -P<YOUR_HADOOP_PROFILE> -D<HADOOP_VERSION> -DskipTests
 ```
 
-Alluxio提供了预定义配置文件，其包含`hadoop-1`，`hadoop-2.2`，`hadoop-2.3` ··· `hadoop-2.9`的Hadoop版本。如果你想编译特定Hadoop版本的Alluxio，你应该在命令中指定版本。
+Alluxio提供了预定义配置文件，其包含`hadoop-2`和`hadoop-3`的Hadoop版本。如果你想编译特定Hadoop版本的Alluxio，你应该在命令中指定版本。
```
Comment (Jackson-Wang-7) [inline docs/cn/ufs/HDFS.md:24]: Alluxio provides predefined build profiles for `hadoop-2` and `hadoop-3` (enabled by default) for the major Hadoop versions 2.x and 3.x.

Comment (Jackson-Wang-7) [reply docs/cn/ufs/HDFS.md:24]: please include the whole sentence

**Comment by Jackson-Wang-7** [comment]
Mostly good to me. Please resolve the conflicts.

**Comment by wuzhenhua01** [comment]
@Jackson-Wang-7 PTAL

**Comment by Jackson-Wang-7** [comment]
@HelloHorizon Please help to merge it

**Review by Jackson-Wang-7 — APPROVED** [review]

**Comment by HelloHorizon** [comment]
alluxio-bot, merge this please

**Pull request merged**